src/palette/internal/palettecreator.cpp

```diff
@@ -828,7 +828,7 @@ PalettePtr PaletteCreator::newOrnamentsPalette(bool defaultPalette)
         auto ornament = Factory::makeOrnament(gpaletteScore->dummy()->chord());
         ornament->setSymId(ornamentType);
         qreal mag = ornament->symId() == SymId::ornamentTrill ? 1.0 : 1.2;
-        sp->appendElement(ornament, ornament->typeUserName(), mag);
+        sp->appendElement(ornament, ornament->translatedTypeUserName(), mag);
```

Comment:
This fix looks not expected to me, and I'm afraid it is not correct. This means that the already-translated name will be stored in the Palette, and thus it will not be retranslated when the user switches to another language.

So, I think it is expected that we pass a TranslatableString to `appendElement`. Instead, you might need to add something in `PaletteCell::translationContext()`, so that the correct context is used when the palette cell calculates its translated name?

Reply:
Hmm, yes, it looked somewhat strange to me too.

Reply:
OK, seems it is 2 changes that are needed